src/components/MessageReactions/MessageReactions.tsx

```diff
@@ -17,8 +30,76 @@ const MessageReactions = () => {
     channel.publish('send', message)
   }
 
+  const sendMessageReaction = (emoji: string, timeserial: any) => {
+    channel.publish('reaction', {
+      body: emoji,
+      extras: {
+        reference: { type: 'com.ably.reaction', timeserial: timeserial },
```

Comment:
Since the variable name is identical, you can simplify this to:
```suggestion
        reference: { type: 'com.ably.reaction', timeserial },
```